smoke-tests/spec/prometheusrules_spec.rb

```diff
@@ -1,39 +0,0 @@
-require "spec_helper"
```

Comment:
And this one as well as above? Even if there is a serviceMonitor, should we not check if the rules are present?

Reply:
new test added